.github/pull_request_template.md

```diff
@@ -0,0 +1,11 @@
+## Description
+<!--Provide high-level overview of what the PR is for.-->
+
+## Implementation
+<!--Add any relevant implementation details that might help the reviewers.-->
```

Comment:
For my edification, could you link to an PR that is a good example of providing implementation details?  I don't presently understand what exactly should go in this section (or if it should be present on most/all PRs)

Reply:
Sure, this is basically to have a chance to explain any implementation details that you think may not be immediately obvious to the reviewer, or help understand the high-level approach to implementation of larger changesets, etc. For example, look at https://github.com/gravitational/gravity/pull/1317 (implementation section is sort of woven into the description there) or https://github.com/gravitational/gravity/pull/1084 or https://github.com/gravitational/gravity/pull/898, etc.